perfkitbenchmarker/vm_util.py

```diff
@@ -246,10 +246,10 @@ def WrappedFunction(*args, **kwargs):
           sleep_time = poll_interval * fuzz_multiplier
           if ((time.time() + sleep_time) >= deadline or
               (max_retries >= 0 and tries > max_retries)):
-            raise e
+            raise
```

Comment:
http://www.ianbicking.org/blog/2007/09/re-raising-exceptions.html

Basically `raise e` gets a stack trace rooted here, whereas `raise` gives you the original stack trace, which is where you actually need to go fix something.